baseclasses/pyAero_solver.py

```diff
@@ -27,7 +27,15 @@ class AeroSolver(BaseSolver):
     """
 
     def __init__(
-        self, name, category={}, def_options={}, informs={}, options={}, immutableOptions=set(), deprecatedOptions={}
```

Comment:
I think changing the order will break pyFriction so we should change that as well.

Reply:
Yeah I will submit PR for the remaining repos that use baseclasses